### PR TITLE
Fixing releasing process - refactoring docker multi-arch building process

### PR DESCRIPTION
**Pull request by rojekp**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -224,8 +222,18 @@ lazy val server: Project = (project in file("server"))
       ),
       publishLocal in Docker := (publishLocal in Docker)
         .dependsOn(yarnTask.toTask(" build"))
-        .dependsOn(createBuildx)
         .value,
+      publish in Docker := {
+        val _ = publishLocal.value
```
Comment (adamw) [inline build.sbt:227]: try: `(publishLocal in Docker).value`

Comment (rojekp) [reply build.sbt]: It works, I had to fix appropriate directory also.

File: build.sbt [hunk 2]
```diff
@@ -305,8 +312,17 @@ lazy val nativeServer: Project = (project in file("native-server"))
     dockerUsername := Some("softwaremill"),
     packageBin in GraalVMNativeImage := (packageBin in GraalVMNativeImage)
       .dependsOn(yarnTask.toTask(" build"))
-      .dependsOn(createBuildx)
       .value,
+    publish in Docker := {
```
Comment (adamw) [inline build.sbt:316]: aren't the above the same? If so, we should extract them to a settings list like `dockerBuildxSettings`

Comment (rojekp) [reply build.sbt]: Ok, extracted.

**Pull request merged**